pkg/provisioner/crdprovisioner.go

```diff
@@ -126,12 +127,36 @@ func (c *CrdProvisioner) CreateVolume(
 	secrets map[string]string,
 	volumeContentSource *v1alpha1.ContentVolumeSource,
 	accessibilityReq *v1alpha1.TopologyRequirement) (*v1alpha1.AzVolumeStatusParams, error) {
+	maxShares := 1
+	var err error
+	if maxSharesStr, ok := parameters["maxShares"]; ok {
+		maxShares, err = strconv.Atoi(maxSharesStr)
+		if err != nil {
+			return nil, status.Error(codes.InvalidArgument, fmt.Sprintf("parse %s failed with error: %v", maxSharesStr, err))
+		}
+		if maxShares < 1 {
+			return nil, status.Error(codes.InvalidArgument, fmt.Sprintf("parse %s returned with invalid value: %d", maxSharesStr, maxShares))
+		}
+	}
+
+	maxMountReplicaCount := maxShares - 1
+	if maxMountReplicaCountStr, ok := parameters["maxMountReplicaCount"]; ok {
```

Comment:
Use `azureutils.MaxMountReplicaCountField` constant.

Key used in search should be lowercase